src/backend/distributed/utils/maintenanced.c

```diff
@@ -235,6 +240,17 @@ InitializeMaintenanceDaemonBackend(void)
 }
 
 
+/*
+ * WarnMaintenanceDaemonNotStarted warns that maintenanced couldn't be started.
```

Comment:
maintenanced -> maintenance

Reply:
This is called maintenanced (short for maintenance daemon)